From 4c5fe9901219956fb754306a2093e78fb460bbdd Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Thu, 23 Jun 2011 10:13:44 +0300
Subject: [PATCH] hostapd: change conf file for better sdk out of box startup

The default hostapd.conf file that is fetched from upstream is using
a band which is not supported by the 1271 chip.
I am using this amend.inc in order to modify it to use g band and default
channel 6 and change the ssid to a more meaningful name.

Upstream-status: inappropriate [SDK specific patch]

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 hostapd/hostapd.conf |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hostapd/hostapd.conf b/hostapd/hostapd.conf
index 6d7263a..bb27beb 100644
--- a/hostapd/hostapd.conf
+++ b/hostapd/hostapd.conf
@@ -83,7 +83,7 @@ ctrl_interface_group=0
 ##### IEEE 802.11 related configuration #######################################

 # SSID to be used in IEEE 802.11 management frames
-ssid=test
+ssid=TexasInstruments_0001

 # Country code (ISO/IEC 3166-1). Used to set regulatory domain.
 # Set as needed to indicate country in which device is operating.
@@ -99,14 +99,14 @@ ssid=test

 # Operation mode (a = IEEE 802.11a, b = IEEE 802.11b, g = IEEE 802.11g,
 # Default: IEEE 802.11b
-hw_mode=a
+hw_mode=g

 # Channel number (IEEE 802.11)
 # (default: 0, i.e., not set)
 # Please note that some drivers (e.g., madwifi) do not use this value from
 # hostapd and the channel will need to be configuration separately with
 # iwconfig.
-channel=60
+channel=6

 # Beacon interval in kus (1.024 ms) (default: 100; range 15..65535)
 beacon_int=100
--
1.7.0.4
